packages/ra-ui-materialui/src/input/ArrayInput/SimpleFormIterator.tsx

```diff
@@ -305,18 +309,13 @@ const Root = styled('div', {
         marginTop: theme.spacing(1),
         [theme.breakpoints.down('md')]: { display: 'none' },
     },
-    [`& .${SimpleFormIteratorClasses.form}`]: {
-        alignItems: 'flex-start',
-        display: 'flex',
-        flexDirection: 'column',
-    },
+    [`& .${SimpleFormIteratorClasses.form}`]: {},
```

Comment:
**suggestion (non-blocking)**: We could remove this so?

```suggestion
    
```

Reply:
No, because even if empty, it's still used by SimpleFormIteratorItem. Users may have used it to apply a custom style to each row. Removing it would be another breaking change we'd need to document.